serf/serf_test.go

```diff
@@ -517,6 +539,179 @@ func TestSerf_leaveRejoinDifferentRole(t *testing.T) {
 	}
 }
 
+func TestSerf_forceLeaveAlive(t *testing.T) {
+	s1Config := testConfig()
+	s2Config := testConfig()
+
+	s1, err := Create(s1Config)
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+	defer s1.Shutdown()
+
+	s2, err := Create(s2Config)
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+	defer s2.Shutdown()
+
+	_, err = s1.Join([]string{s2Config.MemberlistConfig.BindAddr}, false)
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+
+	// s1 should be alive when we call the force leave
+	// but not refute it
+	s2.forceLeave(s1.config.NodeName, true)
```

Comment:
In the other two variations of this test you use `s2` as the node under test (the one you are leaving) and `s1` as the unchanged node, but in this test you flipped the roles around.  For ease of reading you may want to reverse the roles in this test so all of them read the same.

Reply:
fixed!